securedrop_client/gui/auth/__init__.py

```diff
@@ -0,0 +1,20 @@
+"""
+Widgets related to authentication.
+
+Copyright (C) 2018  The Freedom of the Press Foundation.
```

Comment:
Noting that copyright date is 2018, because the header has been naively copied over from the previous files. That's OK, but a larger question is: do we actually need these headers on every source code file? Let's discuss that as a group, preferably with our resident licensing gurus, and maybe we can snip those out for readability's sake.

Reply:
Yes, I'm fixing it in a later PR!